__DEFINES/global.dm

```diff
@@ -320,6 +320,8 @@ var/nanocoins_lastchange = 0
 var/speciesinit = 0
 var/minimapinit = 0
 
+var/bees_species = list()
+
```

Comment:
Should probably be added to the globalaccess helper but it's not super important

Reply:
done

Reply:
sort of but not really, you only added it to `readglobal`, not to `writeglobal` nor to `_all_globals`
the correct way is to run `gen_globals.py`

Reply:
how about now

Reply:
👌